### DISCUSSION TITLE
Reinstalled CasaOS and lost files

### DISCUSSION THREAD

**Discussion by maxzhu143** [IceWhaleTech/CasaOS#1608] [Q&A]
I changed webUI ports and forgot to disable firewall for the ports so I stress re-installed CasaOS with "curl -fsSL https://get.casaos.io | sudo bash file". When I logged back in, all my files were gone but the storage is still pretty full. Is there anyway to recover the files?

**Comment by iaonnic** [comment]
same with me

**Comment by rvvvk** [comment]
I have the same issue, did you find a fix for this @maxzhu143
